.gitignore

```
@@ -42,6 +42,9 @@ latest.dump
 yarn-debug.log*
 .yarn-integrity
 
+# Ignore elasticsearch installations
+/elasticsearch-7.5.2
```

What about ignoring _all_ future versions too?

```suggestion
/elasticsearch*
```

---

@maestromac ah, great call!